rules/17.flooding_get.xml

```diff
@@ -0,0 +1,60 @@
+<beginning>
+<embedded_functions><![CDATA[
+#include "../lib/http2.h"
+#include <stdlib.h>
+
+static unsigned long int number_request = 0;
+
```

Comment:
Please "beautify" the code:

- correct the ident,
- remove unnecessary empty lines
- remove unnecessary printf